docs/api/promotion_coupons.rst

```diff
@@ -48,10 +48,185 @@ If you request for more detailed data, you will receive an object with the follo
+        "used": 0,
+        "createdAt": "@string@.isDateTime()",
+        "updatedAt": "@string@.isDateTime()",
+        "_links": {
+            "self": {
+                "href": "\/api\/v1\/promotions\/HOLIDAY-SALE\/coupons\/1234"
+            },
+            "promotion": {
+                "href": "\/api\/v1\/promotions\/HOLIDAY-SALE"
+            }
+        }
+    }
+
+
+.. warning::
+
+    If you try to create a resource without code, you will receive a ``400 Bad Request`` error.
+
+Example
+^^^^^^^
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/promotions/HOLIDAY-SALE/coupons/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Accept: application/json" \
+        -X POST
+
+Exemplary Response
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    STATUS: 400 Bad Request
+
+.. code-block:: json
+
+    {
+        "code": 400,
+        "message": "Validation Failed",
+        "errors": {
+            "children": {
+                "usageLimit": {},
+                "expiresAt": {},
+                "perCustomerUsageLimit": {},
+                "code": {
+                    "errors": [
+                        "Please enter coupon code."
+                    ]
+                }
+            }
+        }
+    }
+
+You can also create a promotion coupon with additional (not required) fields:
+
++-----------------------+----------------+------------------------------------------------------------------------+
+| Parameter             | Parameter type | Description                                                            |
++=======================+================+========================================================================+
+| Authorization         | header         | Token received during authentication                                   |
++-----------------------+----------------+------------------------------------------------------------------------+
+| promotionCode         | url attribute  | Code of promotion for which the coupon should be created               |
++-----------------------+----------------+------------------------------------------------------------------------+
+| code                  | request        | **(unique)** Promotion coupon identifier                               |
++-----------------------+----------------+------------------------------------------------------------------------+
+| usageLimit            | request        | The information how many times the coupon can be used                  |
++---------------------- +----------------+------------------------------------------------------------------------+
+| perCustomerUsageLimit | request        | The information how many times the coupon can be used by one customer  |
++-----------------------+----------------+------------------------------------------------------------------------+
+| expiresAt             | request        | The information when does the coupon expire                            |
```

Comment:
Not sure, but maybe we should add `*(optional)*` information before all not required fields